oc-chef-pedant/spec/api/status_spec.rb

```diff
@@ -1,12 +1,12 @@
 
-describe "Testing Status API with config", :config do
-  let (:config) { JSON.parse(IO.read("/etc/opscode/chef-server-running.json"))['private_chef'] }
+describe "Testing Status API" do
```

Comment:
The :config tag would let these tests run with the config flag, that might be ok to leave in?

Reply:
I feel it is ok.